code/modules/clothing/suits/cloaks.dm

```diff
@@ -89,3 +89,41 @@
 	heat_protection = HEAD
 	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
 	resistance_flags = FIRE_PROOF | ACID_PROOF
+
+/obj/item/clothing/neck/cloak/skill_reward
+	var/associated_skill_path
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE
+
+/obj/item/clothing/neck/cloak/skill_reward/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>You notice a powerful aura about this cloak, suggesting that only the truly experienced may wield it.</span>"
+
+/obj/item/clothing/neck/cloak/skill_reward/equipped(mob/user, slot)
+	..()
+	if (user.mind?.get_skill_level(associated_skill_path) < SKILL_LEVEL_LEGENDARY)
+		to_chat(user, "<span class = 'notice'>You feel completely and utterly unworthy to even hold \the [src]. You set it on the ground out of respect.</span>")
+		user.dropItemToGround(src, TRUE)
+
+/obj/item/clothing/neck/cloak/skill_reward/gaming
+	name = "legendary gamer's cloak"
+	desc = "Worn by the most skilled professional gamers on the station, this legendary cloak is only attainable by achieving true gaming enlightenment. This status symbol represents the awesome might of a being of focus, commitment, and sheer fucking will. Something casual gamers will never begin to understand."
```

Comment:
doesn't "achieving true gaming enlightenment" only happen after you drink a can of Pwr Game after reaching the legendary gamer skill level? if so, you should make it so that you only receive your legendary gamer's cloak after you open your third eye by doing that

Reply:
while that is true i think i'll leave it as is for consistency